pallets/schemas/src/rpc/src/tests/mod.rs

```diff
@@ -0,0 +1,104 @@
+mod rpc_mock;
+
+use super::*;
+use rpc_mock::*;
+
+use pallet_schemas_runtime_api::SchemasRuntimeApi;
+use std::sync::Arc;
+use substrate_test_runtime_client::runtime::Block;
+
+const SUCCESSFUL_SCHEMA_ID: u16 = 1;
+
+sp_api::mock_impl_runtime_apis! {
```

Comment:
This is the runtime mock. So we are only testing the code in the rpc side of things.